### ISSUE TITLE
As a user, I need offline support (PWA), so I can load pages without an internet connection

### ISSUE THREAD

**Issue by anothermadeline** [rmx-party/choodle#87]
## Criteria

If you want to start a drawing without being connected, that should be possible without an internet connection

(stretch goal for aug 3, in #109): If you want to finish your drawing, it should be possible to save without an internet connection

(not for aug 3): If you want to have multiple drawings (in tabs), you should be able to save them without compromising other drawings in progress.
- Should also have drawings cued to sync to server

Dependent on #24
